File: integration-tests/hibernate-reactive-panache/src/main/java/io/quarkus/it/panache/reactive/NamedQueryWith2QueriesEntity.java
package io.quarkus.it.panache.reactive;

import jakarta.persistence.Entity;
import jakarta.persistence.NamedQuery;

import io.quarkus.hibernate.reactive.panache.PanacheEntity;

@Entity
@NamedQuery(name = "NamedQueryWith2QueriesEntity.getAll1", query = "from NamedQueryWith2QueriesEntity")
@NamedQuery(name = "NamedQueryWith2QueriesEntity.getAll2", query = "from NamedQueryWith2QueriesEntity")
public class NamedQueryWith2QueriesEntity extends PanacheEntity {
    public String test;
}

File: integration-tests/hibernate-reactive-panache-kotlin/src/main/kotlin/io/quarkus/it/panache/reactive/kotlin/NamedQueryWith2QueriesEntity.kt
package io.quarkus.it.panache.reactive.kotlin

import io.quarkus.hibernate.reactive.panache.kotlin.PanacheCompanion
import io.quarkus.hibernate.reactive.panache.kotlin.PanacheEntity
import jakarta.persistence.Entity
import jakarta.persistence.NamedQuery

@Entity
@NamedQuery(name = "NamedQueryWith2QueriesEntity.getAll1", query = "from NamedQueryWith2QueriesEntity")
@NamedQuery(name = "NamedQueryWith2QueriesEntity.getAll2", query = "from NamedQueryWith2QueriesEntity")
class NamedQueryWith2QueriesEntity : PanacheEntity() {
    lateinit var test: String

    companion object : PanacheCompanion<NamedQueryWith2QueriesEntity>
}
